src/Tools/GenerateDocumentationAndConfigFiles/Program.cs

```diff
@@ -589,6 +609,159 @@ static bool checkHelpLink(string helpLink)
                     }
                 }
             }
+
```

Comment:
All the added code here moved as-is from the deleted Program.cs in GenerateGlobalAnalyzerConfigs.csproj.